app/javascript/components/lease_ups/LeaseUpsPage.js

```diff
@@ -1,15 +1,48 @@
 import React from 'react'
 
-import LeaseUpsHeader from './LeaseUpsHeader'
+import PageHeader from '../organisms/PageHeader'
+import PageHeaderSimple from '../organisms/PageHeaderSimple'
 import LeaseUpsTableContainer from './LeaseUpsTableContainer'
 
+const LeaseUpsPageHeader = ({ listing }) => {
+  if (listing) {
+    return (
+      <div>
+        <PageHeader
+          title={listing.Name}
+          content={listing.Building_Street_Address}
+          action={{title: 'Export', link: `/listings/${listing.Id}/lease_ups/export`}}
+          breadcrumbs={[
+            {title: 'Lease Ups', link: '/lease_ups'},
+            {title: listing.Name, link: `/listings/${listing.Id}/lease_ups`}
+          ]} />
+      </div>
+    )
+  } else {
+    return (
+      <div>
+        <PageHeaderSimple title='Lease Ups' />
+      </div>
+    )
+  }
+}
+
 const LeaseUpsPage = ({ listing, results }) => {
-  return (
-    <div>
-      <LeaseUpsHeader listing={listing} />
-      <LeaseUpsTableContainer listing={listing} results={results} />
-    </div>
-  )
+  if (listing) {
```

Comment:
It looks like we are using the same controller for handling LeaseUp for specific listing and all lease ups. Should we have `LeaseUpPage` to handle lease up without listing and and create `ListingLeaseUpPage` to handle lease ups for listings ?

Reply:
After further discussion with @fddayan, it was decided to leave the lease up-related page structure as-is in this PR and let future lease up-specific stories deal with the specifics of lease up pages.